lib/v2/chinacef/index.js

```diff
@@ -0,0 +1,67 @@
+const utils = require('./utils');
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'http://www.chinacef.cn';
+    const regex = /^\/index\.php\/index\/article\/article_id\/\d+/g;
+    const timeRegex = /(\d{4}-\d{2}-\d{2})/g;
+
+    const response = await got({
+        method: 'get',
+        url: rootUrl,
+    });
+    const $ = cheerio.load(response.data);
+
+    const articlesLinkListNode = $('a[target=_black]')
+        .filter((_, item) => item.attribs.href.match(regex))
+        .map((_, item) => item.attribs.href);
+    const articlesLinkList = Array.from(new Set(articlesLinkListNode));
+
+    const out = await Promise.all(
+        articlesLinkList.map(async (item) => {
```

Comment:
```suggestion
        .map((_, item) => item.attribs.href);
        .toArray();

    const out = await Promise.all(
        articlesLinkListNode.map(async (item) => {
```

Reply:
I need to remove the redundant links, I can't use `toArray()` here.